docs/cicero-tutorial_001.md

```diff
@@ -7,6 +7,12 @@ title: Quick Start Tutorial
 
 ## Using an existing Template
 
+### Install Cicero CLI
+
+In order to access the Cicero command line interface (CLI), install the `@accordproject/cicero-cli` npm package:
```

Comment:
Feel free to change this to your preferences but I disagree. In my personal opinion a "Getting started" tutorial should include all necessary steps to get a first example up and running. This should be explained as easily and granular as possible without any references if avoidable.

Reply:
Fair point. I'm just concerned for users who are less technical and new to NPM. I've added a small update to address this. Otherwise your additions remain